src/plugins/validation/2and3/semantic-validators/info.js

```diff
@@ -0,0 +1,40 @@
+// Assertation 1:
+// check if info exists
+
+// Assertation 2:
+// making sure that the required version and title are defined properly
+module.exports.validate = function({ jsSpec }) {
+  const errors = [];
+  const warnings = [];
+  const info = jsSpec.info;
+  if (!info) {
+    errors.push({
+      path: ['info'],
+      message: 'Missing info object for api defintion'
+    });
+  } else if (typeof info != 'object') {
+    errors.push({
+      path: ['info'],
+      message: 'Info is an obect and must have properties'
+    });
```

Comment:
This is looking good. I think it is probably overkill to have both of these as separate checks so let's combine them using something like a `hasInfo` variable:
```js
const hasInfo = info && typeof info === 'object';
```

and then use an error message like "API definition must have an \`info\` object"